mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/41 position the office models

Closes #41 

This just adds a build-in offset to the two office models. This is just a ball-park estimate of where the models should go as I couldn't find any values in the original code that made any sense.

I'm hoping the position is accurate enough for all scenarios.

To test:
- Play any scenario
- Notice the User avatars are somewhat located inside the office spaces correctly.

Notes:
- let me know if there are other office models missing from scenarios (I only have two implemented currently)
- This pull request depends on the #70 PR code for camera control (otherwise you can't see into the offices).

![image](https://user-images.githubusercontent.com/7308321/105411509-55812780-5be8-11eb-941c-46ea748a256b.png)
